components/cam/src/physics/cam/micro_p3_utils.F90

```diff
@@ -192,11 +196,11 @@ subroutine micro_p3_utils_init(cpair,rair,rh2o,rhoh2o,mwh2o,mwdry,gravit,latvap,
     ep_2   = mwh2o/mwdry  ! ratio of molecular mass of water to the molecular mass of dry air !0.622
     rhosur = 100000._rtype/(rd*zerodegc) ! density of air at surface
     rhosui = 60000._rtype/(rd*253.15_rtype)
-    ar     = 841.99667_rtype 
-    br     = 0.8_rtype
-    f1r    = 0.78_rtype
-    f2r    = 0.32_rtype
-    ecr    = 1._rtype
+    ar     = 841.99667_rtype   ! Fallspeed param for rain
+    br     = 0.8_rtype         ! Fallspeed param for for rain
+    f1r    = 0.78_rtype        ! Ventilation param for rain
+    f2r    = 0.32_rtype        ! Ventilation param for rain (note, this is 0.308 in MG)
+    ecr    = 1._rtype          ! Collection efficiency for accretion of rain by snow, AaronDonahue: do we need this?
```

Comment:
I don't see anywhere that actually uses this variable, so no?.?.? Is there a reason why you ask?

Reply:
that is precisely why I ask.  I couldn't find anywhere that used `ecr` either.  I think it is related to the calculation of precip efficiency and is legacy from MG.